backend/schemas.py

```diff
@@ -2,10 +2,11 @@
 from bson.objectid import ObjectId
 
 
-def create_group_object(color, included_documents, label, action, user_id, description, session_id):
+def create_group_object(color, included_documents, label, action, user_id, description, session_id, cluster_id=[]):
     obj = {
         "creation_time": datetime.datetime.utcnow(),
         "sessions": [session_id],
+        "cluster": cluster_id,
```

Comment:
is the type supposed to be an array of OID or just an OID?

Reply:
Yeah I tried just OID but I couldn't figure out how to initialize a group with a null or empty OID. If I initialized a group with `"cluster"=None`, then after when I copied a cluster and assigned `"cluster"=ObjectID(str("oid"))` I would get a type error. the easiest solution was to init as `[]`, then do `[ObjectID(str("oid"))]`, and reset as `[]` on remove.